Quests/Kanto/MoonFossilQuest.lua

```diff
@@ -54,7 +54,8 @@ function MoonFossilQuest:MtMoon1F()
 		return talkToNpcOnCell(19, 16) -- moon stone
 	elseif not self:isTrainingOver() then

```

Comment:
Due to this line, the `self.healPokemonOnceTrainingIsOver` will always return "true" and the bot loops.
Not sure how to fix it tho.